src/blog/2023/11/chatgpt-GPT.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Node-Red Builder a GPT (Alpha) by FlowFuse"
+subtitle: "Speed Up Flow Creation with Your Personal Assistant"
+description: "Speed Up Flow Creation with Your Personal Assistant"
+date: 2023-11-15
+authors: ["grey-dziuba"]
+image: "/blog/2023/10/images/buildvbuy.png"
+tags:
+    - posts
+    - node-red
+    - community
+    - flowfuse
+
+---
+
+When ChatGPT was first released, my expectations were quite low. I had grown accustomed to the usual industry buzz for AI and ML that often led to underwhelming solutions. Naturally, I approached ChatGPT with similar reservations. It wasn't until a few months after its announcement that I decided to give it a try. To my surprise, within just 10 minutes, I found myself so captivated that I decided to purchase the pro version.
+
+<!--more-->
+
+
+On November 6th, OpenAI unveiled a new offering: GPTs. These function as custom ChatGPT environments, allowing the author to provide additional context, giving it a specific and focused purpose. FlowFuse and Node-RED content go hand in hand, making this a perfect application. With new content emerging daily for both platforms, the ability to update and provide essential documentation became increasingly valuable.
```

Comment:
I'm not sure what you mean with the second sentence? It goes hand in hand, sure, but why is this the bridge for the rest of the line?

Reply:
ChatGPT uses old data.  With the announcement of GPTs, we can add our updated documentation for ChatGPT to reference.  I will see if I can clarify.